tools/apkanalyser/README.md

```diff
@@ -0,0 +1,33 @@
+# apk-analyser
+Android APK Analyser especially for Crosswalk/Intel XDK Apps.
+
+# platform
+* Host OS: Linux Ubuntu
+
+# environment
+* Java JDK 1.5 or greater http://www.oracle.com/technetwork/java/javase/downloads/index.html
+* Python 2.7 https://www.python.org/download/
+* Android SDK installed http://developer.android.com
+* Install and set environment for apktool: https://code.google.com/p/android-apktool/wiki/Install
+
+# how to run
+A:<br/>
```

Comment:
`<br/>` sounds not so good. See https://help.github.com/articles/markdown-basics/#ordered-lists